testing/0006-no-ascii-chars

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+source "$TESTS_DIR/functions.sh"
+
+msgfile="$TEST_DIR/message"
+
+cat >"$msgfile" <<EOF
+This is the message with non-ascii characters
```

Comment:
The subject prefix, commit message, commit description, and author are handled by different code paths. Please also use non-ASCII characters in the commit message and author fields to increase code coverage.

Reply:
Sure, good point.

I only put non-ASCII characters in the cover letter (description and subject prefix), I'll follow your suggestion and I'll also add a commit with non-ASCII characters (message, description, author).